Dockerfile

```diff
@@ -0,0 +1,48 @@
+# download the repo: git clone scripts

```

Comment:
not clear what is the main objective of this docker file. It also has some commented code lines.